Utils.swift

```diff
@@ -27,3 +27,23 @@ class Debouncer {
         timer?.invalidate()
     }
 }
+
+// based on https://gist.github.com/tonybruess/9405134
+func isValidGHUsername(uname: String) -> Bool {
+    let len = uname.count
+    if (len < 1 || len > 39) {
+        return false
+    }
+    
+    if uname.first == "-" {
+        return false
+    }
+    
+    for chr in uname {
+        if (chr.isNumber || chr.isLetter || chr == "=") {
```

Comment:
Is it a mistake to use `"="` rather than `"-"` here?

Reply:
Wait a second. Isn't this whole thing broken? This returns true if the first character is valid.

Reply:
Should return true by default and then return false if `!(chr.isNumber || chr.isLetter || chr == "-")` for each character.

Reply:
indeed